addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3635,7 +3634,19 @@ msgctxt "#844"
 msgid "Deactivate"
 msgstr ""
 
-#empty strings from id 845 to 849
+#. Message in a dialog when a user wants to delete a repeating timer
+#: xbmc/pvr/windows/GUIWindowPVRBase.cpp
+msgctxt "#845"
+msgid "Are you sure you want to delete this repeating timer, including all timers it has scheduled?"
```

Comment:
I agree with @ksooo.
I don't like 'repeating timer' and 'timers it has scheduled' much either, but can't think of anything better at the moment. At least it is clear what will happen even if the message is a bit of a mouthful.
Definitely a separate PR to change it, preferably following a thorough forum discussion!